src/js/Navbar.jsx

```diff
@@ -1,28 +1,28 @@
-import React from "react";
-import styled from "styled-components";
-import { Link } from "react-router-dom";
+import React from 'react';
+import styled from 'styled-components';
+import { Link } from 'react-router-dom';
 
-const NavbarDesktopLinks = styled.ul`
+const NavbarDesktopUser = styled.div`
+  position: fixed;
   list-style-type: none;
+  width: -webkit-fill-available;
   margin: 0;
   margin-bottom: 1px;
-  padding: 20px 20px;
+  padding: 3px 30px;
   overflow: hidden;
   background-color: #2f9999;
-
-  li {
-    float: right;
-    margin: 0 auto;
-  }
+  top: 0;
   h2 {
     float: left;
     color: white;
     margin: 0;
   }
-  li a {
-    font-family: "ubuntu";
+  li {
+    float: right;
+    margin: 0 auto;
+    font-family: 'ubuntu';
     display: block;
-    color: white;
+    color: #f7b733;
```

Comment:
Again, any way you can avoid writing out this hex code each time?

Reply:
the quickest next way would be to either write a const somewhere and then export it everywhere, which seems a bit overkill considering it wont be used much more than it already is